api/upload_to_s3.py

```diff
@@ -14,8 +14,8 @@ def get_s3_client(config: _Environ) -> client:
     """Gets S3 client."""
 
     return client('s3',
-                  aws_access_key_id=environ['AWS_ACCESS_KEY_ID'],
-                  aws_secret_access_key=environ['AWS_SECRET_ACCESS_KEY'])
+                  aws_access_key_id=config['AWS_ACCESS_KEY_ID'],
+                  aws_secret_access_key=config['AWS_SECRET_ACCESS_KEY'])
```

Comment:
config is much better if it's being passed into the function, since environ is an actual term. Good to rename